server/player/handler.go

```diff
@@ -83,11 +83,12 @@ type Handler interface {
 	// may be called to cancel the attack, which will cancel damage dealt to the target and will stop the
 	// entity from being knocked back.
 	// The entity attacked may not be alive (implements entity.Living), in which case no damage will be dealt
-	// and the target won't be knocked back.
+	// and the target won't be knocked back, nor will the critical hit particles be displayed.
 	// The entity attacked may also be immune when this method is called, in which case no damage and knock-
 	// back will be dealt.
 	// The knock back force and height is also provided which can be modified.
-	HandleAttackEntity(ctx *event.Context, e world.Entity, force, height *float64)
+	// The attack can be a critical attack
```

Comment:
Could you give more info on what a critical attack does here?

Reply:
Woops, i forgot to do this 😅